docs/content/SRS/SRS.md

```diff
@@ -163,33 +231,83 @@ _The following Software Requirements Specification for `tinymorph` is using [Vol
+
+Rationale: Maintaining consistency in typography ensures readability and prevents user distraction or confusion, contributing to a cohesive user experience.
 
 ### 10.2 Style Requirements
 
-[Insert your content here.]
+LF-S1. The design of the application will be minimalist, utilizing clean lines and a limited color palette.
+
+Rationale: A minimalist design emphasizes functionality and content, enhancing usability by directing the user's focus to essential elements. This approach ensures that the interface remains uncluttered and the features more accessible.
+
+LF-S2. The application must be responsive, adapting seamlessly to various device screens and orientations.
+
+Rationale: As users may access the application from different devices with varying screen sizes, responsiveness is essential to provide a consistent experience across all platforms.
+
+LF-S3. Interactive elements such as buttons and links must contrast significantly with the background to ensure visibility and accessibility.
+
+Rationale: High contrast between elements and backgrounds enhances the visibility of interactive features, making navigation intuitive and preventing user frustration.
+
+LF-S4. The user interface should facilitate smooth transitions and logical navigation pathways between different sections and features of the application.
+
+Rationale: Efficient navigation systems reduce user effort in learning how to use the application and increase overall satisfaction by minimizing the time spent on searching for functionalities.
+
+LF-S5. The application should include visual cues and feedback for user interactions to reinforce usability.
+
+Rationale: Providing immediate visual feedback for user actions confirms the system’s responsiveness, which will help users understand the application’s behavior and reduce errors.
 
 ## 11. Usability and Humanity Requirements
 
 ### 11.1 Ease of Use Requirements
 
-[Insert your content here.]
+UH-EOU1. Tinymorph shall include a session history feature that records and displays the user's most recent editing activities such as document accesses and text modifications.
+
+Rationale: This functionality streamlines user workflow by providing quick access to recent actions, which reduces the time needed for navigation and increases efficiency.
+
+UH-EOU2. The application must allow users to revise or remove their inputs after submission.
+
+Rationale: The ability to alter or delete content post-submission empowers users with greater control over their work, enhancing the accuracy and relevance of their output.
 
 ### 11.2 Personalization and Internationalization Requirements
 
-[Insert your content here.]
+UH-PI1. Tinymorph must include multilingual support to cater to an international audience.
+
+Rationale: Multilingual support enhances the application’s global accessibility and user engagement, ensuring that users can interact with the platform in their preferred language.
```

Comment:
multilingual support ig largely depends on the model support for it as well, so while this can be a requirement, it largely depends on what we need to do for the models.

Reply:
Makes sense, let's leave it for now. Once we confirm what we need to do for the models, we can remove or adjust the requirement accordingly.